GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -648,13 +649,14 @@ private string GetDescribeInfoForRevision(string revision)
         {
             var commitDescribe = _gitDescribeProvider.Get(revision);
```

Comment:
You're free to use it if you think it's clearer. Seems like it would be, for my taste.

Aside, I'd personally avoid using a verb for a variable name.